annileen/engine/renderer.ixx

```diff
@@ -354,7 +354,7 @@ namespace annileen
         }
     }
 
-    void Renderer::renderSceneNode(bgfx::ViewId viewId, ModelPtr model, std::shared_ptr<Material> material)
+    void Renderer::renderSceneNode(bgfx::ViewId viewId, std::shared_ptr<Model> model, std::shared_ptr<Material> material)
```

Comment:
maybe this could also be a const ref, i don't think we change anything in it

Reply:
Yeah, I'm working on it!